views/js/controller/Delivery/monitoring.js

```diff
@@ -434,13 +434,33 @@ define([
                         reasonRequired: true,
                         categoriesSelector: cascadingComboBox(categories[actionName] || {})
                     });
+                    var warningAction;
+                    var warningMessage;
+                    var warningReasons;
 
                     if (!config.allowedResources.length) {
-                        if (_selection.length > 1) {
-                            feedback().warning(__('No report available for these test sessions'));
+                        if (config.deniedResources) {
+                            if (config.deniedResources.length > 1) {
+                                warningAction = __('Cannot') + ' ' + actionName + ' ' + __('these test sessions.');
```

Comment:
This is not well translatable. English may be correct, not the other languages. You should rather use a map of labels, like this:
```javascript
var warningMessages =  {
    pause: __('Cannot pause these test sessions.'),
    terminate: __('Cannot terminate these test sessions.'),
    ...
};
warningAction = warningMessages[actionName];
```